app/scripts/modules/core/delivery/manualExecution/manualPipelineExecution.controller.js

```diff
@@ -115,7 +115,12 @@ module.exports = angular.module('spinnaker.core.delivery.manualPipelineExecution
       addTriggers();
       this.triggerUpdated();
 
-      this.showRebakeOption = pipeline.stages.some((stage) => stage.type === 'bake');
+      let bakeStage = pipeline.stages.find(s => s.type === 'bake');
```

Comment:
In that case, we'll set the initial value to whatever the first bake stage has configured for `rebake`, but the option will apply to all bake stages in the pipeline.

Not sure if there are any pipelines in the wild that are performing multiple bakes with divergent values for the `rebake` field, but if there are, they're probably already confused by the single checkbox option we're presenting.